sdk/go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/hashicorp/go-secure-stdlib/mlock v0.1.3
 	github.com/hashicorp/go-secure-stdlib/parseutil v0.1.8
 	github.com/hashicorp/go-secure-stdlib/password v0.1.1
+	github.com/hashicorp/go-secure-stdlib/permitpool v0.0.0-20250109173936-1ecdd6ad783f
```

Comment:
Should this version not match the top level go.mod version of `v1.0.0`?

Reply:
Great point, will update